Common/SimConfig/include/SimConfig/ConfigurableParam.h

```diff
@@ -105,9 +105,9 @@ class ConfigurableParam
   static void printAllKeyValuePairs();
 
   // writes a human readable JSON file of all parameters
-  static void writeJSON(std::string filename);
+  static void writeJSON(std::string const& filename);
   // writes a human readable INI file of all parameters
```

Comment:
Hi,
I am just wondering why you prefer "std::string const&" to "const std::string&"?
According to this post:
https://stackoverflow.com/questions/3694630/c-const-reference-before-vs-after-type-specifier
the second style is  used by  Stroustrup, in C++ standard itself; and we find it eg. in CPP reference etc.
Thanks.

Reply:
putting const to right to the type is less ambiguous ... I just systematically use it that way out of habbit. In simple cases there is no difference. But already here `const double*` one might not remember what is const here? The double values or the actual pointer?

By putting const to the right one can very well distinguish between
`double const *` and `double * const` because const always applies to the left.

Reply:
Thank you, this makes sense, though it's not habitual.